src/graph.py

```diff
@@ -52,25 +53,27 @@ def edges(self):
         list_key_value = self._dict.items()
         list_edges = []
         for pair in list_key_value:
```

Comment:
A more Pythonic way to do this is:
```python
for key, val in self._dict.items():
    ...
```

This uses "tuple unpacking" to name the tuple items separately for use inside your loop.